app/code/Magento/Catalog/view/adminhtml/web/catalog/product/edit/attribute.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'mage/mage'
+], function (jQuery) {
+    'use strict';
+
+    return function (config, element) {
+
+        jQuery(element).mage('form').mage('validation', {
```

Comment:
Hi @Yogeshks. Thanks for the collaboration. Please use `$` alias to jQuery instead of `jQuery`

Reply:
Hello @VladimirZaets , sure I'll do.